### PR TITLE
HW templates: add a template for Broadcom 5708 NIC

### PR DESCRIPTION
**Pull request by jouvin**

### CODE REVIEW

**Comment by jouvin** [comment]
Sorry, it seems it was my typo of the day! I made several times. Holidays needed 😄

File: hardware/nic/broadcom/bcm5708.pan [hunk 1]
```diff
@@ -0,0 +1,10 @@
+structure template hardware/nic/broadcom/bcm5708;
+
+'model' = 'bcm5709';
```
Comment (jrha) [inline hardware/nic/broadcom/bcm5708.pan]: bcm570~9~**8**?

**Comment by jouvin** [comment]
Hopefully ok now...

**Comment by jouvin** [comment]
@jrha not clear for me too... Not sure we never defined it precisely.

**Pull request merged**